src/styles/sidebar/components/annotation-omega.scss

```diff
@@ -17,4 +19,56 @@
   &__actions {
     margin-left: auto;
   }
+
+  &__form-actions {
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 10px;
+  }
+
+  &__footer {
+    @include var.font-normal;
+    color: var.$grey-5;
+    margin-top: var.$layout-h-margin;
+  }
+
+  &--reply &__footer {
+    margin-top: var.$layout-h-margin - 8px;
+  }
+}
+
+// FIXME vertical rhythm here should be refactored
+.annotation-omega--reply {
+  .annotation-header {
+    // Margin between bottom of ascent of annotation card footer labels
+    // and top of ascent of username should be ~20px
+    margin-top: 0px;
+  }
+
+  .annotation-body {
+    // Margin between top of ascent of annotation body and
+    // bottom of ascent of username should be ~15px
+    margin-top: var.$layout-h-margin - 8px;
+    // Margin between bottom of ascent of annotation body and
+    // top of annotation footer labels should be ~15px
+    margin-bottom: var.$layout-h-margin - 3px;
+  }
+}
+
+.annotation-omega--reply.is-collapsed {
+  margin-bottom: 0;
+
+  .annotation-header {
+    margin: 0;
+  }
+
+  .annotation-body,
+  .annotation-footer {
+    display: none;
```

Comment:
I see this was copied from existing code in `annotation.scss`. I wonder if we could replace this with conditional rendering in JS rather than `display: none` here.

Reply:
Yup; there's a line item for exactly this kind of thing in this follow-up ticket: https://github.com/hypothesis/client/issues/1867 — the mix of descendent selectors, classnames, stateful class names, and in-JS logic for showing and hiding is problematic to be sure!